server/src/controllers/User/resolver.ts

```diff
@@ -49,6 +55,58 @@ export class UserWithInstanceRoleResolver {
     });
   }
 
+  @Query(() => UserInformation)
```

Comment:
```suggestion
  @Query(() => UserInformation, { nullable: true })
```
Unfortunately it's not enough to set the return type, we have to include `nullable: true` so that the schema reflects this.